moc_openshift.py

```diff
@@ -603,17 +635,30 @@ def create_shift_quotas(self, project_name, quota_spec):
             mimetype="application/json",
         )
 
+    def get_resourcequota_details(self, project_name) -> dict:
+        """Returns a list of resourcequota names in the spcified project"""
+        url = f"{self.get_url()}/api/v1/namespaces/{project_name}/resourcequotas"
+        rq_data = self.get_request(url, True).json()
+        self.logger.info("get_resourcequota_details")
+        self.logger.info(pprint.pformat(rq_data))
+        rq_dict = {}
+        if rq_data["kind"] == "ResourceQuotaList":
```

Comment:
If `rq_data["kind"]` is *not* `ResourceQuotaList`, should we log an error? It seems like something has gone very wrong if that's the case.